src/scripts/defaultEvent.js

```diff
@@ -83,7 +86,7 @@ async function notifyUser(evt) {
             if (type == 'break') break
         }
 
-        break
+        // break
```

Comment:
Why this change?

Reply:
So that the dialog stays present even if the user doesn't respond -- I believe it now times out after 15 min.

Reply:
can you delete the line rather than commenting it out?